package.json

```diff
@@ -44,8 +44,8 @@
     "boom": "7.x.x",
     "bounce": "1.x.x",
     "cryptiles": "4.x.x",
-    "hoek": "5.x.x",
-    "joi": "13.x.x",
+    "hoek": "^6.0.1",
```

Comment:
Can you change these back to the .x.x notation?

Reply:
Ah yea, I keep forgetting about this, sry.

Reply:
No problem, just ping me when ready 🎉